### PR TITLE
tikv: deprecate rocksdb's log configs

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1270,18 +1272,30 @@ RocksDB 相关的配置项。
...
 + Info 日志的最大大小。
 + 默认值：1GiB
 + 最小值：0
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1288]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1270,18 +1272,30 @@ RocksDB 相关的配置项。
...
+
 + 日志截断间隔时间，如果为 0s 则不截断。
 + 默认值：0s
 
 ### `info-log-keep-log-file-num`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.file.max-backups`](#max-backups-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1297]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.file.max-backups`](#max-backups-从-v540-版本开始引入) 代替。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1293,6 +1307,10 @@ RocksDB 相关的配置项。
 
 ### `info-log-level`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.level`](#level-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1312]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.level`](#level-从-v540-版本开始引入) 代替。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1846,18 +1864,30 @@ raftdb 相关配置项。
 
 ### `info-log-max-size`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1869]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 代替。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1846,18 +1864,30 @@ raftdb 相关配置项。
...
 + Info 日志的最大大小。
 + 默认值：`"1GiB"`
 + 最小值：`0`
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1880]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1846,18 +1864,30 @@ raftdb 相关配置项。
...
+
 + Info 日志截断间隔时间，如果为 `"0s"` 则不截断。
 + 默认值：`"0s"`
 
 ### `info-log-keep-log-file-num`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.file.max-backups`](#max-backups-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1889]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.file.max-backups`](#max-backups-从-v540-版本开始引入) 代替。
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1869,6 +1899,10 @@ raftdb 相关配置项。
 
 ### `info-log-level`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.level`](#level-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1904]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.level`](#level-从-v540-版本开始引入) 代替。
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -59,6 +59,8 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 
 自 v5.4.0 版本起，废弃原 log 参数 `log-rotation-timespan`，并将 `log-level`、`log-format`、`log-file`、`log-rotation-size` 变更为下列参数，与 TiDB 的 log 参数保持一致。如果只设置了原参数、且把其值设为非默认值，原参数与新参数会保持兼容；如果同时设置了原参数和新参数，则会使用新参数。
 
+自 v5.4.0 版本起，Rocksdb 的 log 也改为由 TiKV 的 log 模块进行管理，Rocksdb 的相关配置项 `info-log-level`、`info-log-max-size`、`info-log-roll-time`、`info-log-keep-log-file-num` 等参数已被废弃不再起作用，相关配置与 TiKV 日志相关配置项合并。注意 Rocksdb 的 `info-log-dir` 依然有效，用于设置 Rocksdb 日志存储的路径。
```
Comment (qiancai) [inline tikv-configuration-file.md:62]: 这里看起来可以删除这句话并把它挪到 v5.4.0 release notes 兼容性变更中 https://docs.pingcap.com/zh/tidb/stable/release-5.4.0#%E5%85%B6%E4%BB%96，这样 L60 那句 “废弃原 log 参数 `log-rotation-timespan`，并将 xxx 变更为下列参数” 和下面的参数看起来更连贯。

**Comment by qiancai** [comment]
@glorv I've added some comments. PTAL.

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1270,18 +1272,30 @@ RocksDB 相关的配置项。
 
 ### `info-log-max-size`
 
+> **警告：**
+>
+> 自 v5.4.0 起，该配置项被废弃，其功能由配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1277]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的 log 模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 代替。
```

**Comment by glorv** [comment]
> @glorv I've added some comments. PTAL.

@qiancai  Done

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1270,18 +1270,30 @@ RocksDB 相关的配置项。
...
 + Info 日志的最大大小。
 + 默认值：1GiB
 + 最小值：0
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.0.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1286]:
```suggestion
> 自 v5.0.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1270,18 +1270,30 @@ RocksDB 相关的配置项。
 
 ### `info-log-max-size`
 
+> **警告：**
+>
+> 自 v5.0.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，其功能由配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 代替。
```
Comment (qiancai) [inline tikv-configuration-file.md:1275]: “自 v5.0.0 起” 要改成 “自 v5.0.0 起” 吗？ log.file.max-size 是 v5.4 引入的

下面的修改也有这个问题

File: tikv-configuration-file.md [hunk 12]
```diff
@@ -1270,18 +1270,30 @@ RocksDB 相关的配置项。
...
 + Info 日志的最大大小。
 + 默认值：1GiB
 + 最小值：0
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1286]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```

File: tikv-configuration-file.md [hunk 13]
```diff
@@ -1846,18 +1862,30 @@ raftdb 相关配置项。
...
 + Info 日志的最大大小。
 + 默认值：`"1GiB"`
 + 最小值：`0`
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃，TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1878]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃。TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```

**Review by qiancai — APPROVED** [review]

File: tikv-configuration-file.md [hunk 14]
```diff
@@ -1270,18 +1270,30 @@ RocksDB 相关的配置项。
...
 + Info 日志的最大大小。
 + 默认值：1GiB
 + 最小值：0
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃。TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1286]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃。TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置按照文件大小自动切分日志的阈值。
```

File: tikv-configuration-file.md [hunk 15]
```diff
@@ -1846,18 +1862,30 @@ raftdb 相关配置项。
...
 + Info 日志的最大大小。
 + 默认值：`"1GiB"`
 + 最小值：`0`
 + 单位：B|KiB|MiB|GiB
 
 ### `info-log-roll-time`
 
+> **警告：**
+>
+> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃。TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置日志自动切分的阈值。
```
Comment (qiancai) [inline tikv-configuration-file.md:1878]:
```suggestion
> 自 v5.4.0 起，RocksDB 的日志改为由 TiKV 的日志模块进行管理，因此该配置项被废弃。TiKV 不再支持按照时间自动切分日志，请使用配置参数 [`log.file.max-size`](#max-size-从-v540-版本开始引入) 配置按照文件大小自动切分日志的阈值。
```

**Pull request merged**